src/runtime/storage/firebase/firebase-storage.ts

```diff
@@ -30,6 +30,8 @@ export async function resetStorageForTesting(key) {
   });
 
   const reference = firebase.database(app).ref(key.location);
+
+  // eslint-disable-next-line no-async-promise-executor
   await new Promise(async resolve => {
     await reference.remove(resolve);
```

Comment:
Given resolve is passed to .remove(), I don't think we need async + await on the promise executor.

Reply:
done